exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/advanced.ipynb

```diff
@@ -0,0 +1,342 @@
+{
```

Comment:
<p>"you encounter a problem <strong>when running</strong> one of the other notebooks ..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Fixed</p>